src/govee_ble/parser.py

```diff
@@ -26,6 +26,9 @@
 PACKED_hhhhh = struct.Struct(">hhhhh")
 
 
+UNAVAILABLE = "unavailable"
```

Comment:
```suggestion
ERROR = "error"
```

Reply:
Lets call it error because thats what it is

Reply:
https://github.com/home-assistant/core/blob/8819634b613f6bfd55885283bab86c3852ae40c4/homeassistant/components/govee_ble/sensor.py#L126

Reply:
There where we run into the typing issue

Reply:
I think it would still be ok if we override the `available` property in that class to return False if `self.processor.entity_data.get(self.entity_key) == ERROR`

Reply:
As suggested: https://github.com/PierreAronnax/home-assistant-core/commit/b2d7445748d9b653d254763e3a7813db33b5928c
Needs test.